framework/core/js/src/forum/utils/alertEmailConfirmation.js

```diff
@@ -59,7 +59,7 @@ export default function alertEmailConfirmation(app) {
     }
   }
 
-  m.mount($('<div/>').insertBefore('#content')[0], {
+  m.mount($('<div class="Alert--emailConfirmation"/>').insertBefore('#content')[0], {
     view: () => (
       <ContainedAlert dismissible={false} controls={[<ResendButton />]}>
```

Comment:
Agree with @davwheat. How about:

```suggestion
  m.mount($('<div class="App-notices"/>').insertBefore('#content')[0], {
    view: () => (
      <ContainedAlert dismissible={false} controls={[<ResendButton />]} className="Alert--emailConfirmation">
```

Reply:
This seems like a better approach, imo.

Reply:
@rob006 this should do it for your use case while still sticking to our conventions, any objection to making the change and merging?

Reply:
HTML structure is fine, but I'm not sure how it would scale. If we would have another type of alert, does it mean we would have multiple `div.App-notices` containers?

Reply:
No there should always be the one only `App-notices` container, unless an extension is really set on hacking that.

For 2.0 it will most likely be made extensible to easily have multiple possible notices as we've seen it to be a plausible use case.